### DISCUSSION TITLE
Unable to swap connection styles

### DISCUSSION THREAD

**Discussion by iNDiGOxNOiSE** [jgraph/drawio#4480] [General]
Made an account to drop this here, but for some reason I'm unable to swap between connection styles (line, link, arrow, etc.) from the drop down menu given. I can open the menu, but when I click on whichever style I want it stays locked into "Line". The other drop down menus, like "Waypoints", works fine. I have tried reloading the project, restarting my PC, etc. to no avail. Just looking for this to be fixed, thanks :)
![image](https://github.com/jgraph/drawio/assets/172850076/24a8cc6d-4a79-4f08-89c5-58025bc079d4)

Little recording of the exact issue:
https://gyazo.com/3be13208fd8788808764543952b84476

**Comment by davidjgraph** [comment]
Fix in 24.5.5.
